script.js

```diff
@@ -0,0 +1,168 @@
+// DOM 조작(검색)
+const form = document.querySelector('#input-form');
+const input = document.querySelector('#input-form input');
+const yetList = document.querySelector('#yet-list');
+const doneList = document.querySelector('#done-list');
+const todoNum = document.querySelector('#todoNum');
+const doneNum = document.querySelector('#doneNum');
+
+// 텍스트는 변수로 설정 => 오타 방지 및 수정 용이
+const yetText = 'yetList';
+const doneText = 'doneList';
+const yetBtnText = 'yetBtn';
```

Comment:
저는 이럴 경우 변수명과 문자열 값을 똑같이 설정하는 게 더 좋을 것 같습니다. 코드 리뷰하면서 이게 은근 헷갈리더라구요...!

```suggestion
const yetList = 'yetList';
const doneList = 'doneList';
const yetBtn = 'yetBtn';
```

Reply:
오오 감사합니다 그렇게 해야겠어요